test/verify/cracklib.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of Cockpit.
```

Comment:
Does this have anything to do with the very well-known library of the same name?